yarn.lock

```diff
@@ -1773,6 +1778,23 @@
   resolved "https://registry.yarnpkg.com/@next/swc-win32-x64-msvc/-/swc-win32-x64-msvc-12.3.4.tgz#d28ea15a72cdcf96201c60a43e9630cd7fda168f"
   integrity sha512-DQ20JEfTBZAgF8QCjYfJhv2/279M6onxFjdG/+5B0Cyj00/EdBxiWb2eGGFgQhrBbNv/lsvzFbbi0Ptf8Vw/bg==
 
+"@noble/curves@1.0.0", "@noble/curves@~1.0.0":
+  version "1.0.0"
+  resolved "https://registry.yarnpkg.com/@noble/curves/-/curves-1.0.0.tgz#e40be8c7daf088aaf291887cbc73f43464a92932"
+  integrity sha512-2upgEu0iLiDVDZkNLeFV2+ht0BAVgQnEmCk6JsOch9Rp8xfkMCbvbAZlA2pBHQc73dbl+vFOXfqkf4uemdn0bw==
+  dependencies:
+    "@noble/hashes" "1.3.0"
+
+"@noble/hashes@1.3.0":
+  version "1.3.0"
+  resolved "https://registry.yarnpkg.com/@noble/hashes/-/hashes-1.3.0.tgz#085fd70f6d7d9d109671090ccae1d3bec62554a1"
+  integrity sha512-ilHEACi9DwqJB0pw7kv+Apvh50jiiSyR/cQ3y4W7lOR5mhvn/50FLUfsnfJz0BDZtl/RR16kXvptiv6q1msYZg==
+
+"@noble/hashes@~1.3.0":
+  version "1.3.1"
+  resolved "https://registry.yarnpkg.com/@noble/hashes/-/hashes-1.3.1.tgz#8831ef002114670c603c458ab8b11328406953a9"
+  integrity sha512-EbqwksQwz9xDRGfDST86whPBgM65E0OH/pCgqW0GBVzO22bNE+NuIbeTb714+IfSjU3aRk47EUvXIb5bTsenKA==
```

Comment:
Unfortunately, this doesn't work because `viem` has the version pinned to `1.3.0`. I can manually overwrite it through yarn's resolutions, but that'd require a change in the root `package.json`: 

https://stackoverflow.com/questions/54120149/within-a-yarn-workspace-how-can-i-force-a-resolution-for-a-child-projects-depe

And also not sure about overwriting a crypto-related dependency. It should be fine in this particular case, but it still feels wrong

Reply:
Here's full log of why the package is here:
```
yarn why v1.22.19
[1/4] Why do we have the module "@noble/hashes"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "@noble/hashes@1.3.1"
info Has been hoisted to "@noble/hashes"
info Reasons this module exists
   - "workspace-aggregator-a318d789-96b7-4978-a887-465d6d113a72" depends on it
   - Hoisted from "_project_#@coinbase#wallet-sdk#@solana#web3.js#@noble#hashes"
   - Hoisted from "_project_#@coinbase#wallet-sdk#@solana#web3.js#@noble#curves#@noble#hashes"
   - Hoisted from "_project_#@web3-react#gnosis-safe#@safe-global#safe-apps-sdk#viem#@scure#bip32#@noble#hashes"
   - Hoisted from "_project_#@web3-react#gnosis-safe#@safe-global#safe-apps-sdk#viem#@scure#bip39#@noble#hashes"
info Disk size without dependencies: "1.05MB"
info Disk size with unique dependencies: "1.05MB"
info Disk size with transitive dependencies: "1.05MB"
info Number of shared dependencies: 0
=> Found "@noble/curves#@noble/hashes@1.3.0"
info This module exists because "_project_#@noble#curves" depends on it.
info Disk size without dependencies: "1.05MB"
info Disk size with unique dependencies: "1.05MB"
info Disk size with transitive dependencies: "1.05MB"
info Number of shared dependencies: 0
=> Found "viem#@noble/hashes@1.3.0"
info This module exists because "_project_#@web3-react#gnosis-safe#@safe-global#safe-apps-sdk#viem" depends on it.
info Disk size without dependencies: "1.05MB"
info Disk size with unique dependencies: "1.05MB"
info Disk size with transitive dependencies: "1.05MB"
info Number of shared dependencies: 0
Done in 0.26s.
```

Reply:
Also, the package is just [234 bytes](https://bundlephobia.com/package/@noble/hashes@1.3.1), so it shouldn't bloat the bundle size